NetCasbin/CoreEnforcer.cs

```diff
@@ -406,11 +472,12 @@ public bool Enforce(params object[] requestValues)
         /// <param name="requestValues">The request needs to be mediated, usually an array of strings, 
         /// can be class instances if ABAC is used.</param>
         /// <returns>Whether to allow the request and explains.</returns>
-        public async Task<(bool Result, IEnumerable<IEnumerable<string>> Explains)>
-            EnforceExAsync(params object[] requestValues)
+#if !NET45
+        public (bool Result, IEnumerable<IEnumerable<string>> Explains)
+            EnforceEx(params object[] requestValues)
         {
             var explains = new List<IEnumerable<string>>();
-            bool result = await EnforceAsync(requestValues, explains);
```

Comment:
if the original code is `await EnforceAsync`, how can it pass build, since the method is sync